ozaria/site/views/play/level/tome/CastButtonView.coffee

```diff
@@ -75,21 +73,10 @@ module.exports = class CastButtonView extends CocoView
     castRealTimeShortcutVerbose = (if @isMac() then 'Cmd' else 'Ctrl') + '+' + @castShortcutVerbose()
     castRealTimeShortcutVerbose + ': ' + $.i18n.t('keyboard_shortcuts.run_real_time')
 
-  onCastButtonClick: (e) ->
-    Backbone.Mediator.publish 'tome:manual-cast', {realTime: false}
+  onUpdateButtonClick: (e) ->
+    Backbone.Mediator.publish 'tome:recompile'
 
-  onCastRealTimeButtonClick: (e) ->
-    if @options.level.get('replayable') and (timeUntilResubmit = @options.session.timeUntilResubmit()) > 0
-      Backbone.Mediator.publish 'tome:manual-cast-denied', timeUntilResubmit: timeUntilResubmit
-    else
-      Backbone.Mediator.publish 'tome:manual-cast', {realTime: true}
-    @updateReplayability()
-
-  onClickGameDevPlayButton: ->
-    Backbone.Mediator.publish 'tome:manual-cast', {realTime: true}
-
-  onDoneButtonClick: (e) ->
-    return if @options.level.hasLocalChanges()  # Don't award achievements when beating level changed in level editor
+  onNextButtonClick: (e) ->
     @options.session.recordScores @world?.scores, @options.level
     Backbone.Mediator.publish 'level:show-victory', { showModal: true, manual: true }
```

Comment:
You will need to pass `capstoneInProgress: true/false` here for the victory modal as we discussed offline.